test/views/components/gds/MainTemplateSpec.scala

```diff
@@ -60,7 +59,7 @@ class MainTemplateSpec extends SpecBase with ViewMatchers {
         val view: Document = createView(withFileUploadFlag = true)
 
         val scripts = view.getElementsByTag("script").asScala.toSeq
-        val scriptAttribs = scripts.map(_.attr("src"))
+        val scriptAttribs = scripts.map(_.attr("src").replaceFirst("^/cds-file-upload-service", ""))
```

Comment:
This is not clear to me. What is the intent of this change?

Reply:
It's weird. I have seen a different behaviour running the tests from the command-line (`sbt test`) compare to running them by using Intellij, at least on my PC.
Running them with sbt, `/cds-file-upload-service` is prepended to `/assets/javascripts/...`, while this does not take place with Intellij. It would be good if one of you could try it too.